### ISSUE TITLE
how to specify path for generation app.html 

### ISSUE THREAD

**Issue by rustam87** [gaven/nuxt-svg-sprite-module#5]

**Comment by gaven** [comment]
Hi @rustam87 - you can't at the moment. A typical Nuxt app reads the `app.html` at the root of the directory. If you don't mind me asking, what is your use case?

**Comment by rustam87** [comment]
my nuxt app placed in src folder, but app.html generated in '../src'
i found option where i can specify the path to app.html
Thanks

```
 build: {

    templates: [{
      type: 'app',
      src: './app.html',
    }],
```

**Comment by gaven** [comment]
@rustam87 I wonder if I could get the module to read from the template config. For example it defaults to `./` unless there is a template src defined in your `nuxt.config.js`

**Comment by rustam87** [comment]
this variant not working. SVG not visible Sorry. I need manualy transfer app.html in src folder

**Comment by gaven** [comment]
@rustam87 That makes sense. I'll push a fix later today that should solve your issue.

**Comment by rustam87** [comment]
HI, thanks. I'm waiting this feature

**Comment by gaven** [comment]
This has been fixed in #6
